- # [Research](https://www.barkerlab.net/research)
  collapsed:: true
	- Questions #meeting_november_8
		- How does polyploidy influence rates of adaptation and speciation?
		- Do changes in the efficacy of selection in polyploids lead to higher rates of adaptation or impede evolution relative to diploids?
		- How may the population genetics of polyploid species influence their macroevolutionary outcomes?
		- Why do polyploid species diploidize if they were successful as polyploids?
		- Does natural selection or drift drive diploidization?
		- Are there universal rules that govern genome evolution following genome duplication?
		- What are the relative roles of natural selection and drift driving the retention and loss of genes following polyploidy?
		- Does dosage-balance constrain the evolution of large fractions of paleopolyploid genomes?
		- How do multiple origins of polyploid species contribute to polyploid success?
		- Does a single genome organization emerge from multiple origins?
		- If so, does selection or drift drive the evolution of genome organization from multiple origins?
		- Are polyploids of different origins interfertile?
		- How does hybridization influence the evolutionary success of polyploids?
		- Do allopolyploids or autopolyploids have an evolutionary advantage?
		- Does hybridization (and polyploidy) itself create evolutionary novelty or does simply cutting off gene flow from the broader species range allow rare genotypes to persist and proliferate?
		- Why do rates of chromosome gain and loss vary dramatically across the phylogeny?
		- How do these rates impact diploidization following polyploidy?
		- What evolutionary forces drive the gain and loss of chromosomes, and ultimately the evolution of chromosome number?
- # Informatics
  collapsed:: true
	- The links on the informatics side of things are broken #meeting_november_8
	- The evo-pipes repo is perl heavy.
	- The following text is there:
		- We also actively develop and maintain a suite of evolutionary bioinformatic tools to facilitate our research. Our tools are hosted at our public facing server, [EvoPipes.net](http://evopipes.net/). We are currently developing new bioinformatic approaches for identifying and analyzing ancient whole genome duplications and hybridization. If you are interested in software or bioinformatic algorithms, we would be happy to have you join us!
	- How is your compute environment set up?
- # [Study systems](https://www.barkerlab.net/systems)
  collapsed:: true
	-
	- #meeting_november_8 I am not the most versed in taxonomy but this description of "triangle of U" of Brassica is fascinating.
	- The plants of *Brassica* are well known as the “dogs” of the plant world because of their morphological diversity. A series of nested polyploid speciation, or whole genome duplication, events occurred in the ancestry of these plants. We are currently studying how polyploidy contributed to the diversity of the *Brassica *crops (e.g., turnip, Chinese cabbage, canola oil, broccoli, cauliflower, kale, cabbage, etc.). These crops are a model agricultural system for understanding how changes in natural selection and genetic variation following polyploidy may drive diversity.
- # Courses
	- The list of courses is probably not exhaustive #meeting_november_8
		- Bioinformatics
		- Evolutionary genetics
		- Genome evolution